runtime/compiler/z/codegen/S390J9CallSnippet.cpp

```diff
@@ -1718,3 +1720,183 @@ TR::S390InterfaceCallReadOnlySnippet::getLength(int32_t estimatedSnippetStart)
            + sizeof(int32_t)  /*RIP offset to PICSlot data address*/
            + sizeof(int32_t);  /*RIP offset to instruction in mainline*/
    }
+ 
+uint8_t *
+TR::S390J9CallSnippetRX::emitSnippetBody()
+   {
+   uint8_t *cursor = cg()->getBinaryBufferCursor();
+   TR::SymbolReference *callSymRef = getRealMethodSymbolReference();
+   TR::Node *callNode = getNode();
+   getSnippetLabel()->setCodeLocation(cursor);
+   cursor = S390flushArgumentsToStack(cursor, getNode(), getSizeOfArguments(), cg());
+
+   //Check if the call is unresolved
```

Comment:
This comment is not very useful. It does not say anything more than the code already does.

Reply:
Removed in https://github.com/eclipse/openj9/compare/9a9d9b37daeb983c8d4e295fcc785afeecde5d78..de725983cd06ff96468a76b30321d3f8bb48caea